app/views/bids/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Bids#index</h1>
```

Comment:
These were auto-generated. Maybe we should pay attention to rails g commands and what they are creating!